packages/providers/src/provider.ts

```diff
@@ -843,16 +843,17 @@ export default class Provider {
   }
 
   async getResourcesForTransaction(
-    owner: AbstractAddress,
+    owner: string | AbstractAddress,
```

Comment:
Could a `AddressLike` type be better here as a single point of entry?

Reply:
It won't matter after this:
 - https://github.com/FuelLabs/fuels-ts/issues/1584